### PR TITLE
Added gender for automatic retrieval.

### PR DESCRIPTION
**Pull request by jamesbigmac**

### CODE REVIEW

File: srcalc/src/main/java/gov/va/med/srcalc/web/view/VariableEntry.java [hunk 1]
```diff
@@ -61,13 +61,12 @@ public static VariableEntry withRetrievedValues(final Collection<? extends Varia
         	{
         		continue;
         	}
-            // TODO: Exchange the switch statement for a better solution.
+        	// TODO: Exchange the switch statement for a better solution.
```
Comment (jamesbigmac) [inline srcalc/src/main/java/gov/va/med/srcalc/web/view/VariableEntry.java]: Ummm, what? I'm guessing Eclipse had something to do with this.

File: srcalc/src/main/java/gov/va/med/srcalc/util/VistaFieldTranslation.java [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * This class is devoted to translating any information that comes from Vista
+ * to element values present on the web pages (values that come from the srcalc database).
+ */
+public class VistaFieldTranslation {
```
Comment (dgtombs) [inline srcalc/src/main/java/gov/va/med/srcalc/util/VistaFieldTranslation.java]: I think putting these constants in `VistaPatientDao` would be better encapsulation. Want to discuss?

File: srcalc/src/main/java/gov/va/med/srcalc/vista/RpcVistaPatientDao.java [hunk 3]
```diff
@@ -43,7 +44,7 @@ public Patient getPatient(final int dfn)
 	        final String[] fieldArray = results.get(0).split("\\^");
 	        final String patientName = fieldArray[0];
 	        final int patientAge = Integer.parseInt(fieldArray[1]);
-	        final String patientGender = fieldArray[2];
+	        final String patientGender = VistaFieldTranslation.TRANSLATION_MAP.get(fieldArray[2]);
```
Comment (dgtombs) [inline srcalc/src/main/java/gov/va/med/srcalc/vista/RpcVistaPatientDao.java]: `get()` will return null if the map doesn't have a value. Should we handle that somehow?

File: srcalc/src/main/java/gov/va/med/srcalc/web/view/VariableEntry.java [hunk 4]
```diff
@@ -61,13 +61,12 @@ public static VariableEntry withRetrievedValues(final Collection<? extends Varia
...
             switch(v.getRetrievalKey())
-            {	
-//	            case 1:
-//	        		variableEntry.fDynamicValues.put(v.getKey(), patient.getGender()
-//        				.equalsIgnoreCase("M")? "male": "Female");
-//	        		break;
+            {
+	            case 1:
+	        		variableEntry.fDynamicValues.put(v.getKey(), patient.getGender());
```
Comment (dgtombs) [inline srcalc/src/main/java/gov/va/med/srcalc/web/view/VariableEntry.java]: Mmmm these lines are tantalizingly similar to case 2 now, but I don't think we can improve it yet.

**Pull request merged**